### PR TITLE
Back button font family and font size

### PR DESCRIPTION
**Pull request by yogevbd**
Closes #5922

### CODE REVIEW

File: playground/ios/Podfile.lock [hunk 1]
```diff
@@ -1,14 +1,14 @@
 PODS:
```
Comment (ItsNoHax) [inline playground/ios/Podfile.lock]: These Podfile changes look unrelated to the merge request?

**Comment by guyca** [comment]
@yogevbd Can you please update  `OptionsTopBarBackButton` in Options.ts?
